doc/source/conf.py

```diff
@@ -92,6 +92,9 @@
 
 myst_heading_anchors = 3
 
+nitpicky = True
+nitpick_ignore_regex = [("py:class", ".*")]
```

Comment:
what do these mean?

Reply:
See the sphinx documentation for info on what these do: https://www.sphinx-doc.org/en/master/usage/configuration.html.

In short, they make broken references into build time errors. Broken class references are ignored because type annotations have a [history of raising errors of this type](https://github.com/search?q=repo%3Asphinx-doc%2Fsphinx+annotation+nitpick&type=issues), and AFAICT there's not an elegant way of solving them right now save for having a big list of custom type annotations to ignore.